app/Entities/Response.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Entities;
+
+readonly class Response
+{
+    public function __construct(
+        private string $statusCodeHeader,
```

Comment:
1. Why header suffix? I think statusCode would be sufficient
2. Pass enum itself not value
3. There are problems with implementation that I will comment on LocalServer.php